contracts/utils/SafeCast.sol

```diff
@@ -106,4 +106,16 @@ library SafeCast {
         require(value >= 0, "SafeCast: value must be positive");
         return uint256(value);
     }
+
+    /**
+     * @dev Converts an unsigned uint256 into a signed int256.
+     *
+     * Requirements:
+     *
+     * - input must be less than or equal to maxInt256.
+     */
+    function toInt256(uint256 value) internal pure returns (int256) {
+        require(value < 2**255, "SafeCast: value doesn\'t fit in 255 bits");
```

Comment:
```suggestion
        require(value < 2**255, "SafeCast: value doesn't fit in an int256");
```

While technically correct, I think this wording will make it clearer to users as to what is going on. Also, you don't need to escape the ' :)

Reply:
Done!